src/main/java/org/jboss/remoting3/remote/RemoteConnectionProvider.java

```diff
@@ -204,7 +208,32 @@ public Cancellable cancel() {
         if (useSsl && destination instanceof InetSocketAddress) {
             if (xnioSsl == null) {
                 try {
-                    xnioSsl = xnio.getSslProvider(connectOptions);
+                    Boolean usePKCS = connectOptions.get(RemoteConnectionProviderFactory.JBOSS_AS_REMOTE_USEPKCS, false);
```

Comment:
We don't want to introduce a bunch of new options which will be immediately obsolete in WildFly 11 and EAP 7.1.